sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -116,6 +119,16 @@ export interface AttestationResult {
     version?: string;
 }
 
+// @public (undocumented)
+export class AttestationSigner {
```

Comment:
If this class is going to be part of the convenience layer, I think the class itself and every member should get documented.

Reply:
100% agreed.